docs/Intro/dive/rwaOracle.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 4
+description: Chronicle RWA Oracle
+keywords: [RWA Oracle, Real World Assets]
+---
+# Chronicle RWA Oracle
+## What are RWAs?
+
+In the context of blockchain, Real-World Assets (RWAs) refer to physical assets and traditional financial instruments represented as digital assets on a blockchain network. These assets can range from assets such as real estate, commodities, art, and even intellectual property, to financial assets such as bonds, stocks, or ETFs . RWAs provide a bridge between traditional and decentralized finance opening up new opportunities for market inclusion, accessibility, and innovation.
+
+
+
+<div style={{textAlign: 'center'}}>
+<img
+    src="/img/Intro/Dive/rwa.png"
+    alt="Example of Connect to Web3 button"
+ 
+/>
+</div>
+
+## Chronile Protocol Initial RWA Integration with M^0
+In August 2024, Chronicle launched its Real World Asset (RWA) Oracle with an integration with [M^0](https://www.m0.org/), a decentralized onchain protocol, as well as a set of off-chain standards and APIs, that allows multiple Minters to issue a fully fungible cryptodollar called $M.
```

Comment:
> launched its Real World Asset

How about "Launched a Real World Asset?". I'm note sure "its" is good choice of words bc we want to deploy many more rwa oracles. wdyt?